maintenance_plan/__manifest__.py

```diff
@@ -1,13 +1,12 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {'name': 'Maintenance Plan',
  'summary': 'Extends preventive maintenance planning',
- 'version': '10.0.1.0.0',
- 'author': 'Odoo Community Association (OCA), Camptocamp SA',
+ 'version': '11.0.1.0.0',
+ 'author': 'Odoo Community Association (OCA), Camptocamp SA, Appcademy SRL',
```

Comment:
I don't think `Appcademy SRL` should be mentioned as "Author" here, as the contribution was a basic porting of the module. The Contributors list was already updated and IMO that would be enough.

Reply:
I agree. Please put it on contributors only

Reply:
Done!